docs/source/tutorials/raptor.ipynb

```diff
@@ -169,10 +169,17 @@
    },
    "outputs": [],
    "source": [
+    "# function for raptor to execute\n",
+    "@rp.pythontask\n",
+    "def msg(val: int):\n",
+    "    if(val %2 == 0):\n",
+    "        print('Regular message')\n",
+    "    else:\n",
+    "        print(f'This is a very odd message: {val}')\n",
+    "\n"
```

Comment:
This line is missing a comma.  Hmm, how did that happen?  Strange...

Reply:
This is my fault. Sorry for not actually running the notebook myself. Not it is fixed.